initrd/bin/config-gui.sh

```diff
@@ -514,6 +524,34 @@ while true; do
         fi
       fi
     ;;
+    "K" )
+      if [ "$CONFIG_USER_USB_KEYBOARD" != "y" ]; then
+        if (whiptail --title 'Enable USB Keyboard?' \
+             --yesno "USB keyboards will be usable in $CONFIG_BRAND_NAME.
+                    \n\nIgnoring USB keyboards prevents a compromised USB device from controlling
```

Comment:
Maybe rephrase "Ignoring ... prevents" to "Enabling ... allows", to avoid confusing positive/negative answer here. But otherwise yes, such warning when enabling is okay.

> probably removing x230-hotp-maximized_usb-kb 

Where is the config stored, in /boot? If so, for CI I may still need to have build time option then, because some tests involve wiping the whole disk clean, so the config won't survive (and then test cannot continue as USB input is no longer accepted).

Reply:
That's a good thought, I think I like that wording better for the 'enable' prompt.  I'll give that a try.

I have that text in both the 'enable' and 'disable' prompts at the moment, I thought users would wonder why it was a setting at all (especially non-Qubes users).  But given that it's still off by default, maybe we don't need it in the 'disable' prompt, you would have seen the 'enable' text already.  I'll try only showing it on enable.

>Where is the config stored, in /boot?

It's the same as the other config menu settings, it's in config.user in flash.  So it will survive erasing /boot, but it will not survive a 'flash erasing settings' unless you insert a config.user in that flash image ahead of time (which works, I do it to offer preconfigured "basic" images and such).

Reply:
> So it will survive erasing /boot

All good then, in that case x230-hotp-maximized_usb-kb could be dropped (at least for my needs).

Reply:
@marmarek as of this PR, x230-hotp-maximized_usb-kb will stay. But all other boards having native ps2 keyboard support will permit to enable usb keyboard support to be enabled from Config menu, save settings and warn that reboot is required (leading to TOTP/HOTP resealing and TPM DUK if enabled) since firmware is modified).

@JonathonHall-Purism Once message to user is changed on enablement, good to merge!

Reply:
That change is up, thanks for patience as I had to prioritize a few other things before I could circle back to these PRs :bow: